app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/Product/CollectionProcessor/AttributeProcessor.php

```diff
@@ -39,7 +40,8 @@ public function __construct($fieldToAttributeMap = [])
     public function process(
         Collection $collection,
         SearchCriteriaInterface $searchCriteria,
-        array $attributeNames
+        array $attributeNames,
+        ContextInterface $context
     ): Collection {
```

Comment:
we're gonna have to add to all of these modified classes suppress for unused parameters
best to do the interface optional and not pass this when not needed